desktop/cmp/contextmenu/ContextMenu.ts

```diff
@@ -55,12 +55,12 @@ function parseItems(items: MenuItemLike[]): ReactNode[] {
     });
 
     return items
-        .filter(it => isMenuItem(it) && !it.hidden && !it.omit)
+        .filter(it => !isMenuItemCfg(it) || (!it.hidden && !it.omit))
```

Comment:
This logic here and in mobile Menu was stripping out strings and elements, meaning the code on lines 62-63 was never used.